h/security/policy/top_level.py

```diff
@@ -78,4 +69,22 @@ def get_subpolicy(request):
         secure=request.scheme == "https",
     )
     html_authcookie = html_authcookie.bind(request)
+
+    if is_api_request(request):
+        return APIPolicy(
+            [
+                BearerTokenPolicy(),
+                AuthClientPolicy(),
+                APICookiePolicy(
+                    # Use html_authcookie rather than api_authcookie to
+                    # authenticate API requests, at least for now. This is a
+                    # hopefully temporary workaround for an undiagnosed issue
+                    # we've been seeing in production, see:
+                    # https://hypothes-is.slack.com/archives/C2BLQDKHA/p1725041696040459
+                    html_authcookie,
```

Comment:
The diff is confusing but they key change is that is used to be `APICookiePolicy(api_authcookie, ...)` and now it's `APICookiePolicy(html_authcookie, ...)`. This changes `APICookiePolicy` (which is the security policy that enables certain API requests to be authenticated using cookies) to accept the original HTML auth cookie instead of the new API auth cookie.